aspnetcore/signalr/version-differences.md

```diff
@@ -67,6 +69,10 @@ ASP.NET Core SignalR now supports [streaming data](xref:signalr/streaming) from
 
 The ability to pass arbitrary state between clients and the hub (often called HubState) has been removed, as well as support for progress messages. There is no counterpart of hub proxies at the moment.
 
+### PersistentConnection Deprecation
```

Comment:
If the class doesn't exist in Core, it's not deprecated, it just doesn't exist and that's a difference between the two, not deprecation.

It's still viable in 4.x SignalR unless it's deprecated there in which case the deprecation comment is not about Core.

Reply:
Makes sense. Changed the terminology - let me know if further changes are needed. Thanks!